README.md

```diff
@@ -1,10 +1,14 @@
-# Islandora Altmetrics [![Build Status](https://travis-ci.org/Islandora/islandora_altmetrics.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_altmetrics)
+# Islandora Badges[![Build Status](https://travis-ci.org/Islandora/islandora_altmetrics.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_altmetrics)
```

Comment:
Repository is still named Altmetrics, even though the module is Badges, for this round, correct? I'm not sure how this part works.